bin/prettier.js

```diff
@@ -112,7 +112,7 @@ if (stdin) {
         output = format(input);
       } catch (e) {
         process.exitCode = 2;
-        console.error(e);
+        console.error(filename + ": " + e);
```

Comment:
This is also really nice: You get to see _which_ file errored, when multiple files are passed.